addons/mail/static/src/messaging/component/thread_unread_preview/thread_unread_preview.js

```diff
@@ -10,7 +10,7 @@ const mailUtils = require('mail.utils');
 
 const { Component } = owl;
 
-class ThreadPreview extends Component {
+class ThreadUnreadPreview extends Component {
```

Comment:
I'm not sure how I feel about this rename, but it also contains thread that are actually read...

Reply:
Maybe keep `ThreadPreview` name for this one, and use `ThreadNeedactionPreview` for the newly introduced one?

We are fully aware of problems regarding functional intentions of messaging menu, so very likely to be redesigned after brainstorm with PO's, that's why I don't mind having `ThreadPreview` and `ThreadNeedactionPreview` at the moment.

Reply:
Yes this is what I suggested, keep `ThreadPreview` unchanged, and the new one whatever the name, `ThreadNeedactionPreview` is fine I don't think I suggested something else.